chain/store/name.go

```diff
@@ -68,7 +67,7 @@ func (sdb *StateDB) getRegistrant(name string) ([]byte, uint32, error) {
 	nameId := getNameId(name)
 	ni, err := sdb.getNameInfo(nameId)
 	if err != nil {
-		return nil, 0, errors.New("name registration info not found")
+		return nil, 0, fmt.Errorf("name registration info not found, %v", err)
```

Comment:
We can simply return err here because `getNameInfo` error is not always not found error